doc/Type/Backtrace.pod6

```diff
@@ -77,16 +77,16 @@ Defined as:
 Returns a summary string representation of the backtrace, filtered
 by C<!.is-hidden && (.is-routine || !.is-setting)>.
 
+    this program:
```

Comment:
de-indent or comment this?

Reply:
First version was based on example [debugging docu](https://docs.perl6.org/programs/01-debugging#___top) but I did change to the current version as requested. Is there a good example somewhere already, how to document this?

Reply:
Indented text is going to be treated as a program. Just de-indent it.

Reply:
But without indentation all text will be rendered on the same line in think.

Reply:
Just that first line, as above.

Reply:
(as a matter of fact, I don't think the initial version was incorrect, but if you do it this other way, please try and take that into account)